src/content/faq/faq.mdx

```diff
@@ -1,6 +1,7 @@
 ---
 title: Frequently Asked Questions
 description: Review frequently asked questions about the GSA SmartPay program. Learn about state taxes, program coordinator responsibilities, merchants, government shutdowns, and more.
+intro: "Review frequently asked questions (FAQs) about the GSA SmartPay® program."
```

Comment:
It looks like the intention here is to add this intro text to the FAQ page. But I think we will need to make an edit to the .astro document to actually pull this in. I can't find this text on the page.